plugins/org.eclipse.elk.core/src/org/eclipse/elk/core/Core.melk

```diff
@@ -384,12 +384,24 @@ group spacing {
         targets parents, nodes
     }
 
-    advanced option individualOverride: IndividualSpacings {
-        label "Individual Spacing Override"
-        description
-            "In general spacing values apply to the children of the hierarchical node 
-            (possibly the root node) for which the values are actually specified. Hereby,
-            the children include ports, edges, and labels. "
+    advanced option individual: IndividualSpacings {
+        label "Individual Spacing"
+        description 
+            "Allows to specify individual spacing values for graph elements that shall be different from 
+            the value specified for the element's parent."
+        documentation
+            "In most cases, spacing values apply to the children of the hierarchical node 
+            (possibly the root node) for which the values are actually specified. 
+            Hereby, the children may include ports, edges, and labels.
```

Comment:
This particular sentence might be misleading. To affect the spacing of node labels, an individual override has to be set on the node. Specifying a node label spacing on the node would not work. Or am I confused?